biocrnpyler/core/chemical_reaction_network.py

```diff
@@ -47,7 +47,7 @@ def species(self):
     def species(self, species):
         if not hasattr(self, "_species"):
             self._species = []
-            self._species_dict = {}
+            self._species_set = set()
```

Comment:
It would be useful to document the change of the species dict object to a set object in the PR description, so that we can get more information about why this change was made.

Reply:
Thanks for pointing this out. I added the following to the docstring:
        A _species_set is used to ensure that no duplicate species are added to the CRN.
        In earlier BioCRNPyler versions, a _species_dict with key as species, and value as "True" was used, which was unintuitive.
        Since sets are designed to keep unique elements, so we use a set to keep track of species.